iceoryx_posh/source/roudi/roudi_cmd_line_parser.cpp

```diff
@@ -60,6 +63,18 @@ void CmdLineParser::parse(int argc, char* argv[], const CmdLineArgumentParsingMo
             m_run = false;
             break;
 
+        case 'u':
+        {
+            uint16_t roudiId{0u};
+            if (!cxx::convert::fromString(optarg, roudiId))
+            {
+                LogError() << "The RouDi id must be in the range of [0, " << ((1 << 16) - 1) << "]";
```

Comment:
Could we have a `constexpr` here instead of shifts?

Reply:
done